deptofdefense/move.mil
# Upgrade application Ruby version to 2.4

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…


## Summary of Changes

This pull request updates the project to use Ruby version 2.4.2. This will match our upcoming staging and production environments and generally keeps us up-to-date with the latest Ruby.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout ruby-upgrade`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rake`,
1. run `bin/rubocop`, and
1. run `bin/brakeman`.
